_layouts/usecases.html

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+---
+<div class="card gx-5 gy-5 m-5">
+    <div class="card-header">
+        <div class="card-title display-6 mx-2 mt-3 mb-4">Use Cases</div>
+        <ul class="nav nav-tabs nav-justified fs-5 card-header-tabs" id="usecases" role="tablist">
+            <li class="nav-item">
+                <button class="nav-link active" id="encryption-at-rest-usecase" data-bs-toggle="tab" data-bs-target="#encryption-at-rest-usecase-tab-pane" type="button" role="tab" aria-controls="encryption-at-rest-usecase-tab-pane" aria-selected="true">
```

Comment:
I'm not sold on having each use case as a tab. I can see merit in it, and is clearly OK to cover the two we have for 0.5.0. I think we do want to include some sort of overview /table of contents (toc) type thing.

Reply:
I switch to jekyll [collection](https://jekyllrb.com/docs/collections/).  Currently that's being rendered to a single page, but it would be easy to switch to n pages.  (I wonder if quickstarts should by a collection too?).